wikimedia/wikipedia-ios
# Article as a Living Document - Article Content Insert piece

**Fixes Phabricator ticket:** 

Epic with original mocks: https://phabricator.wikimedia.org/T241253
Subticket with additional business logic in comment discussions: https://phabricator.wikimedia.org/T257595

### Remaining Tasks

Nothing major, shouldn't block overall code review at this point.

- [x] Uncomment unit tests and fix
- [x] Add bot icon for bots
- [x] Fix small bug where some articles are slightly horizontally scrollable
- [x] Fix bug with new changes badge logic being overwritten

### Test Steps
1. You should be able to see the article content insert on LTR devices in any EN Wikipedia article for iOS13+
